app/addons/replication/__tests__/helpers-tests.js

```diff
@@ -0,0 +1,34 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
```

Comment:
@timmak I prefer something like that. I see we just have to edit the [testRegex](https://facebook.github.io/jest/docs/configuration.html#testregex-string). Since all our current tests have `Spec` i them. Any tests we want jest to run should have `.test` in them. I think that could be nicer than moving folders etc.